1.0.0-beta4/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mono:3.12
```

Comment:
I tried it with debian:wheezy but it gave me a compile error. Probably I'm missing something. Let me try again and otherwise mono:4 makes more sense.